grammar.js

```diff
@@ -285,6 +286,7 @@ module.exports = grammar({
     // -------------------------------------------------------------------------
 
     syntax_quote: $ => seq('`', $._anything),
+    var_quote: $ => seq('#\'', $._anything),
```

Comment:
I think this may need to be `seq("#'", $.symbol),`

I'm not sure var quote works with anything other than symbols. I'm not 100% on this though, maybe @shaunlebron can help confirm that?

Reply:
the reader just wraps it in `(var ...)` regardless of the type, but [`var`](http://cljs.github.io/api/cljs.core/var)

Reply:
Cool, guys! So I should change it to:
```suggestion
    var_quote: $ => seq("#'", $.symbol),
```

Is that right?

Reply:
This looks correct 👍 

Sorry for the delay in response; Shaun and I are both at Clojure/conj and not on our computers too much ;)